Public/Src/Cache/DistributedCache.Host/Configuration/LocalCasSettings.cs

```diff
@@ -51,7 +52,8 @@ public LocalCasSettings()
                 grpcPort: grpcPort,
                 grpcPortFileName: grpcPortFileName,
                 bufferSizeForGrpcCopies: bufferSizeForGrpcCopies,
-                gzipBarrierSizeForGrpcCopies: gzipBarrierSizeForGrpcCopies);
+                gzipBarrierSizeForGrpcCopies: gzipBarrierSizeForGrpcCopies,
+                grpcPoolSize: grpcPoolSize);

```

Comment:
Should it be "grpcThreadPoolSize", otherwise it is hard to tell what this thing actually configures...

Reply:
I see that this is the name that was used everywhere before, but I think in the global config it make sense to use a more descriptive name (I would actually vote for changing the name everywhere, but this is up to you).

---
In reply to: [333632905](https://github.com/microsoft/BuildXL/pull/1026#discussion_r333632905) [](ancestors = 333632905)